### PR TITLE
Add presentational prop to NxThreatIndicator - RSC-1107 

### PR DESCRIPTION
**Pull request by rjp99**
https://issues.sonatype.org/browse/RSC-1107

### CODE REVIEW

File: lib/src/components/NxThreatIndicator/NxThreatIndicator.tsx [hunk 1]
```diff
@@ -29,12 +29,17 @@ const NxThreatIndicator: FunctionComponent<Props> =
         className = classnames(classNameProp, `nx-threat-indicator nx-threat-indicator--${category}`),
         label = `threat level ${category}`;
 
-    return (
+    const icon = (
+      <NxFontAwesomeIcon aria-hidden={false}
```
[the remark this answers is not in the document]
Comment (EmilyBermudez88) [reply lib/src/components/NxThreatIndicator/NxThreatIndicator.tsx:33]: After reading up on the presentation role for aria, makes sense. Thanks for the clarification

File: gallery/src/components/NxThreatIndicator/NxThreatIndicatorPage.tsx [hunk 2]
```diff
@@ -68,6 +68,14 @@ const NxThreatIndicatorPage = () =>
             <NxTable.Cell>No</NxTable.Cell>
             <NxTable.Cell>Additional CSS class names to apply to the rendered element</NxTable.Cell>
           </NxTable.Row>
+          <NxTable.Row>
+            <NxTable.Cell>presentational</NxTable.Cell>
+            <NxTable.Cell>boolean</NxTable.Cell>
+            <NxTable.Cell>No</NxTable.Cell>
+            <NxTable.Cell>
+              When this is set to true the tooltip will be hidden and the icon's role will be set to "presentation"
```
Comment (rpokorny) [inline gallery/src/components/NxThreatIndicator/NxThreatIndicatorPage.tsx:76]: I think this should describe when/why you'd want to do this

File: lib/src/components/NxThreatIndicator/NxThreatIndicator.tsx [hunk 3]
```diff
@@ -29,12 +29,17 @@ const NxThreatIndicator: FunctionComponent<Props> =
...
+                         aria-label={label}
```
[the remark this answers is not in the document]
Comment (entroform) [reply lib/src/components/NxThreatIndicator/NxThreatIndicator.tsx]: > This should not be provided when the role is presentation

Fixed: ee15c8834c07a9dde42786cb07a03fd09a46ed12

File: lib/src/components/NxThreatIndicator/__tests__/NxThreatIndicator.test.tsx [hunk 4]
```diff
@@ -94,4 +94,22 @@ describe('NxThreatIndicator', function() {
     expect(icon).toHaveClassName('nx-threat-indicator');
     expect(icon).toHaveClassName('nx-threat-indicator--low');
   });
+
+  it('should hide tooltip when presentational prop is set true', function() {
+    const tooltip = getShallowComponent({ presentational: true }).find(NxToolip);
+
+    expect(tooltip).not.toExist();
```
Comment (rpokorny) [inline lib/src/components/NxThreatIndicator/__tests__/NxThreatIndicator.test.tsx:101]: Might want to assert that the icon is still there

File: lib/src/components/NxThreatIndicator/NxThreatIndicator.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (entroform) [reply lib/src/components/NxThreatIndicator/NxThreatIndicator.tsx]: > Why are we explicitly setting `aria-hidden` at all?

Removed it here: ee15c8834c07a9dde42786cb07a03fd09a46ed12

File: lib/src/components/NxThreatIndicator/__tests__/NxThreatIndicator.test.tsx (hunk 4 above)
Comment (entroform) [reply lib/src/components/NxThreatIndicator/__tests__/NxThreatIndicator.test.tsx:103]: > Might want to assert that the icon is still there

Added it here: 94dceaf99a0ca4ae5d4afb8bc05b2ab3d8553bae

**Pull request merged**